code/datums/supplypacks/supply.dm

```diff
@@ -178,8 +178,8 @@
 					)
 	cost=25
 	containertype = /obj/structure/closet/crate/secure/xion
-	containername = "Explorer equipment"
-	access = access_explorer
+	containername = "Away Team equipment"
+	access = access_eva
```

Comment:
Turning this into list of (access_eva, accses_awayteam) may be a good option considering the name. It does work for supplypacks.